packages/xo-server-backup-reports/src/index.js

```diff
@@ -679,6 +679,11 @@ class BackupReportsXoPlugin {
           status: nagiosStatus,
           message: nagiosMarkdown,
         }),
+      xo.sendIcinga2Status !== undefined &&
+        xo.sendIcinga2Status({
+          status: nagiosStatus === 0 ? 'OK' : 'CRITICAL',
+          message: markdown,
```

Comment:
Did the markdown displayed well on Nagios?

Reply:
You mean Icinga2 ?
I figured nagiosMarkdown was a reduced text specific for nagios.